src/sentry/api/endpoints/organization_config_integrations.py

```diff
@@ -2,14 +2,26 @@
 
 from rest_framework.response import Response
 
-from sentry import integrations
+from django.conf import settings
+
+from sentry import integrations, features
 from sentry.api.bases.organization import OrganizationEndpoint
 from sentry.api.serializers import serialize, IntegrationProviderSerializer
 
 
 class OrganizationConfigIntegrationsEndpoint(OrganizationEndpoint):
     def get(self, request, organization):
-        providers = list(integrations.all())
+        has_catchall = features.has(
+            "organizations:internal-catchall", organization, actor=request.user
```

Comment:
This feature flag has been problematic in the past as it required more work to do alpha/early adopter groups. You might want to add an explicit feature flag for pagerduty.

Reply:
Yeah I was going to add a new flag later down the road once its more fleshed out - but just did this so I didn't have to add a new flag right away. Could be an easy follow up PR